matter/wifi/wfx_notify.cpp

```diff
@@ -34,90 +34,120 @@ using namespace ::chip::DeviceLayer;
  * Notifications to the upper-layer
  * All done in the context of the RSI/WiFi task (rsi_if.c)
  */
-/*
- * Device started..
- */
+
+/***********************************************************************************
+ * @fn  wfx_started_notify()
+ * @brief
+ *       Wifi device started notification
+ * @param[in]: None
+ * @return None
+ *************************************************************************************/
 void wfx_started_notify()
 {
   sl_wfx_startup_ind_t evt;
   sl_wfx_mac_address_t mac;
 
   EFR32_LOG("%s: started.", __func__);
 
-  memset(&evt, 0, sizeof evt);
+  memset(&evt, CLEAR_BUFFER, sizeof evt);
   evt.header.id     = SL_WFX_STARTUP_IND_ID;
   evt.header.length = sizeof evt;
   evt.body.status   = 0;
   wfx_get_wifi_mac_addr(SL_WFX_STA_INTERFACE, &mac);
-  memcpy(&evt.body.mac_addr[0], &mac.octet[0], 6);
+  memcpy(&evt.body.mac_addr[0], &mac.octet[0], MAC_ADDRESS_FIRST_OCTET);
 
   PlatformMgrImpl().HandleWFXSystemEvent(WIFI_EVENT, (sl_wfx_generic_message_t *)&evt);
 }
 
-/*
+/***********************************************************************************
+ * @fn  void wfx_connected_notify(int32_t status, sl_wfx_mac_address_t *ap)
+ * @brief
  * For now we are not notifying anything other than AP Mac -
  * Other stuff such as DTIM etc. may be required for later
- */
+ * @param[in] status:
+ * @param[in] ap: access point
+ * @return None
+ *************************************************************************************/
 void wfx_connected_notify(int32_t status, sl_wfx_mac_address_t *ap)
 {
   sl_wfx_connect_ind_t evt;
 
   EFR32_LOG("%s: started.", __func__);
 
-  if (status != 0) {
+  if (status != SUCCESS_STATUS) {
     EFR32_LOG("%s: error: failed status: %d.", __func__, status);
     return;
   }
 
   EFR32_LOG("%s: connected.", __func__);
 
-  memset(&evt, 0, sizeof evt);
```

Comment:
You can leave the 0 there.

Rule of the thumb for defines is to ask the following question : "Will I have to explain where this number comes from ?" if the answer is yes, add a define, if the answer is no don't add one. 

In this case you're clearly clearing a buffer, so no need to replace the `0` with a define

Reply:
Missing brackets `sizeof(evt)`

Reply:
removed CLEAR_BUFFER where it is used  and added brackets around sizeof